test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/SuppressWarningsInMembersAndTypes.cs

```diff
@@ -128,4 +130,20 @@ public static void Test ()
 		[RequiresUnreferencedCode ("")]
 		static void MethodWithRUC () { }
 	}
+
+	class SuppressOnProperty
+	{
+		public static void Test ()
+		{
+			var test = Property;
+		}
+
+		[UnconditionalSuppressMessage ("Test", "IL2072")]
+		static int Property {
```

Comment:
Can you add a property and an event (once events are supported too) in the SuppressWarningsInType class just to also test that we can go through the path of "I had a method, went to the property/event didn't find a suppression, so I went to the declaring type and I found it there"